x-pack/qa/rolling-upgrade-multi-cluster/src/test/java/org/elasticsearch/upgrades/CcrRollingUpgradeIT.java

```diff
@@ -244,8 +245,8 @@ public void testBiDirectionalIndexFollowing() throws Exception {
                 case NONE -> {
                     createLeaderIndex(leaderClient(), "leader_index5");
                     index(leaderClient(), "leader_index5", 128);
-                    followIndex(followerClient(), "leader", "leader_index5", "follower_index5");
-                    followIndex(leaderClient(), "follower", "follower_index5", "follower_index6");
+                    followIndexIgnoringBadRequests(followerClient(), "leader", "leader_index5", "follower_index5");
```

Comment:
Wouldn't an easier alternative be to put a `ensureGreen(leaderClient(), "leader_index5");` before?

Reply:
Ensure green won't guarantee that the node (from the remote cluster) that will coordinate the underlying index stats request will be on the latest cluster state - hence the retries

Reply:
FTR I think it would, because `ensureGreen` maps to a `GET _cluster/health` with `?wait_for_events=LANGUID` which means it waits for all in-flight cluster state updates to complete, including being applied on all nodes (or time out, but that's not expected in an integ test like this).